docs/using/sdk_example.md

```diff
@@ -0,0 +1,4 @@
+<h1></h1>
```

Comment:
This weirdness is because MkDocs *really* wants to put a page title here, but there's a titile header in the notebook. I tried a number of other options, and this was the cleanest I could come up with